deepfence_utils/postgresql/migrate/0006_create_generative_ai_integration_table.sql

```diff
@@ -1,10 +1,13 @@
 -- +goose Up
 
 -- +goose StatementBegin
-CREATE TABLE public.ai_integration
+DROP TABLE IF EXISTS ai_integration;
```

Comment:
there is no need for this as the code was not released

Reply:
I wanted to clean up developer setup, but yes, it won't cause any issue, so can be removed.